backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/flightleg/BpkFlightLeg.kt

```diff
@@ -73,13 +77,19 @@ fun BpkFlightLeg(
     ) {
         Row(modifier = Modifier.fillMaxWidth()) {
             carrierLogoContent?.let {
-                Box(
-                    modifier = Modifier.padding(top = BpkSpacing.Sm),
-                    content = it,
-                )
+                Box(modifier = Modifier.padding(top = BpkSpacing.Sm)) {
```

Comment:
i think you could've just applied the padding before the size and then it treats it as a margin instead

Reply:
yes good point, forgot about it! applied. thanks!